awx/main/tasks/jobs.py

```diff
@@ -723,16 +561,16 @@ def run(self, pk, **kwargs):
                 if not params[v]:
                     del params[v]
 
-            self.dispatcher = CallbackQueueDispatcher()
+            self.runner_callback.dispatcher = CallbackQueueDispatcher()
```

Comment:
I can't see any reason for `CallbackQueueDispatcher` to be in this file. I think it should be moved to the callback module. It should work in the `__init__` method.

Reply:
I have made the required changes.